Source/WebCore/platform/graphics/displaylists/DisplayListItems.h

```diff
@@ -1834,6 +1835,7 @@ class DrawControlPart {
 private:
     Ref<ControlPart> m_part;
     FloatRect m_rect;
+    FloatRoundedRect m_borderRect;
```

Comment:
Do we need both m_rect and m_borderRect? Is m_borderRect.rect() different from m_rect?

Reply:
No they should be the same. `RenderStyle::getRoundedBorderFor()` adds the radii if the `RenderStyle::hasBorderRadius()` is true. So I removed the `FloatRect` argument.